### PR TITLE
LPS-149738 - Include border color utility classes in Dialect

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:relevant

**Comment by ilzamcmed** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js [hunk 1]
```diff
@@ -29,6 +29,95 @@ const BORDERS = [
 	'rounded-pill',
 ];
 
+const BORDER_PRIMARY = [
```
Comment (ethib137) [inline modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js:32]: Can you move all of this to the colors tab instead of the general tab. Make sure you are following the same pattern as what is set for the background colors.

File: modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/mixins/_colors.scss [hunk 2]
```diff
@@ -6,4 +6,8 @@
 	.text-#{$class-suffix} {
 		color: $color !important;
 	}
+
+	.border-#{$class-suffix} {
```
Comment (ethib137) [inline modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/mixins/_colors.scss:10]: Let's sort whenever possible.

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -2319,7 +2319,12 @@ boost-must-be-greater-than-or-equal-to-0=Boost must be greater than or equal to
 bootstrap-mlockall=Bootstrap Mlockall
 bootstrap-mlockall-help=This configuration only applies to EMBEDDED (Development) mode. If this is checked, the process address space is locked into RAM, preventing any Elasticsearch memory from being swapped out.
 border=Border
+border-accent-colors=Border Accent Colors
```
Comment (ethib137) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:2322]: Add all your new language keys to the end of the file, and let the sorting be done by the buildLang task in a separate commit. This will make it much easier to handle rebasing when necessary.

**Review by ethib137 — CHANGES_REQUESTED** [review]

File: modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/mixins/_colors.scss (hunk 2 above)
Comment (ilzamcmed) [reply modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/mixins/_colors.scss:10]: Fixed

File: modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js:32]: Fixed

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 3 above)
Comment (ilzamcmed) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:2322]: Thanks for the hint, Evan!

File: modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js [hunk 4]
```diff
@@ -85,7 +85,6 @@ const GeneralGuide = () => {
 					<TokenItem className={item} key={item} label={item} />
 				))}
 			</TokenGroup>
-
```
Comment (ethib137) [inline modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js:88]: There shouldn't be any changes to this file now.

**Review by ethib137 — CHANGES_REQUESTED** [review]
After these changes it should be ready to be forwarded. Before that you'll want to update the commit history. I think the best way to do the following:

- Create a new branch off of your existing one. 
- You can then clean up the commit history, making the changes we've discussed. 
- Push your new branch to your remote.
- Send a new PR to me.
- Ask for a PR review.
- Run ci:test:sf and ci:test:relevant

I'll review and if everything looks right I'll forward it to BChan.

If you have any questions about this process we can discuss it in our meeting today.

@eduardobbgf FYI, general process once review is almost complete.
